### ISSUE TITLE
Workshop Single Template

### ISSUE THREAD

**Issue by coreymckrill** [WordPress/Learn#21]
![workshop-single](https://user-images.githubusercontent.com/916023/89588444-7f8c8c00-d7f8-11ea-81ef-614ab1366817.jpg)

https://www.figma.com/file/UtmG8KQv3Csy8hNtuAWkwZ/Learn-WordPress?node-id=15%3A1390

**Comment by StevenDufresne** [comment]
The only thing missing here, after #30 is merged, is that the blue navigation bar has 3 links `Lesson Plans, Workshops, Workshop Ideas`. 

That isn't implemented at the moment. It also goes beyond the scope of this specific page. We could probably open a new ticket for it and close this one.

**Comment by coreymckrill** [comment]
Cool, I created a new issue for that, so closing this.
